script/gen_requirements_all.py

```diff
@@ -38,6 +38,7 @@
 TEST_REQUIREMENTS = (
     'aioautomatic',
     'aiohttp_cors',
+    'aiohttp_session',
```

Comment:
Perhaps I should also be adding cryptography here, but the travis tests have passed, so maybe not....